esy-build/EsyRc.re

```diff
@@ -35,9 +35,14 @@ let ofPath = path => {
 
   let filename = Path.(path / ".esyrc");
 
-  if%bind (Fs.exists(filename)) {
-    ofFile(filename);
-  } else {
-    return(empty);
+  let filenameInHome = Path.(Path.homePath() / ".esyrc");
+
+  let%bind filenameExists = Fs.exists(filename);
+  let%bind filenameInHomeExists = Fs.exists(filenameInHome);
```

Comment:
I'm not sure how to run this only when we need to

Reply:
(on a phone) you can do check in else branch (which was there previously)

Reply:
Didn't get that to work before but now it worked